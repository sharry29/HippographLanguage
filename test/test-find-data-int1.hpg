int main() {
	graph<string:int> g = ["a":10 -()> "b":20 <()> "c":20; "a" <()- "d"];
	graph<string:int> g_new = g.find(20);
	for_node(n : g_new) {
		n.print();
		print("");
	}
}