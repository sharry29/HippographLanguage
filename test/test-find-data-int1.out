"b":20
"c":20
